deepchem/feat/dft_featurizer.py

```diff
@@ -0,0 +1,332 @@
+from __future__ import annotations
+import os
+from abc import abstractmethod, abstractproperty
+from typing import List, Dict, Optional, Union
+import numpy as np
+import torch
+import dqc
+from dqc.system.mol import Mol
+from dqc.system.base_system import BaseSystem
+from dqc.grid.base_grid import BaseGrid
+from deepchem.utils.dftutils import KSCalc
+
+class System(dict):
```

Comment:
We should have a more descriptive name; System can mean many things and we should be precise. Perhaps AtomicSystem? or DFTSystem?